pySDC/implementations/problem_classes/FastWaveSlowWave_0D.py

```diff
@@ -13,27 +13,17 @@ class swfw_scalar(ptype):
     Attributes:
     """
 
-    def __init__(self, problem_params, dtype_u=mesh, dtype_f=imex_mesh):
+    dtype_u = mesh
+    dtype_f = imex_mesh
+
+    # TODO : choose default values
```

Comment:
did you check if there are some parameters in the paper that we could use for defaults? I guess @pancetta would know.

Reply:
You mean for `lambda_s`, `lambda_f` and `u0`, yes?

Reply:
If so, then I'd suggest
`lambda_s = -1`
`lambda_f = -1000`
`u0 = 1`
if that actually works..